### PR TITLE
Store data

### PR DESCRIPTION
**Pull request by PromzzyKoncepts**
# Hey There,
> In this PR, we successfully

- [x]  Stored data entered from books, rentas, and person in json files
- [x] Ran test to confirm they were preserved
- [x] Refactored the code
- [x] Committed and  pushed changes
- [x] Corrected linters

### CODE REVIEW

File: main.rb [hunk 1]
```diff
@@ -2,38 +2,9 @@
...
-    create_rental
-  when '6'
-    list_all_rentals
-  else
-    puts 'Please enter a number between 1 and 7.'
-  end
-end
-
-main
+puts main
```
Comment (andgarzonmal) [inline main.rb:10]: - [ ] Please check your app is working correctly, I'm getting this error when running your app, so I'm unable to check the behavior of your project.

image

**Review by andgarzonmal — CHANGES_REQUESTED** [review]
# ♻️♻️♻️ STATUS: Changes Required  ♻️♻️♻️


 Hi @PromzzyKoncepts ✋🏼 

#### Your project is excellent. You are almost done. However, there are some issues you should fix. Please fix them and your project will be approved.

___ 

### 💯  Highlights 💯
- Your gitHub flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️

### ♻️ Request changes ♻️

- [x] _Check the comments under the review._
___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

### Cheers and Happy coding! 💻 👏
almost


___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by alexander16108 — CHANGES_REQUESTED** [review]
Hi @PromzzyKoncepts @andgarzonmal , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.



## Invalid Code Review Request

You have submitted a project, that has failing linters 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: rental.rb [hunk 2]
```diff
@@ -7,8 +7,10 @@ class Rental
 
   def initialize(date, book, person)
     @date = date
+
     @book = book
     book.rental << self
+
     @person = person
     person.rental << self
   end
```
Comment (Munsa1) [inline rental.rb:16]: - [ ] When the user tries to list all rentals for a given ID, the following error occurs and the program is terminated, kindly fix this.
image

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @Team

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
Passing Linters :heavy_check_mark: 
Correct GitHub Flow :heavy_check_mark: 
Descriptive ReadMe File :heavy_check_mark: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by PromzzyKoncepts** [comment]
# Hello, Reviewer

**Please if you note any error or something that needs fixed, kindly post a comment here or send me a message on slack**
>> Promise Okechukwu

**Review by tugsem — APPROVED** [review]
## STATUS: APPROVED 👍🎯  
Hi @PromzzyKoncepts , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⭐ 
✔️ Great work making the changes!
 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**